landoapi/landing_worker.py

```diff
@@ -227,18 +227,12 @@ def extract_error_data(exception: str) -> tuple[list[str], list[str]]:
             re.MULTILINE,
         )
 
-        # RE to capture files that could not be merged for one reason or another.
```

Comment:
If I understand correctly, we have some error messages interspersed with some confirmation/successful messages, and our workaround to fix the problem is to stop parsing for both. Could we (or do we plan to) re-enable this parsing and account for the potential "successfully applied hunk" error message?

Reply:
Yes - I'll file a bug for this. That particular functionality would be needed in order to determine that cause of the conflict, but not specifically to find the failed paths (which can be done just by looking at the .rej paths).